spec/unit/burndown_chart_spec.rb

```diff
@@ -492,12 +492,21 @@
   end
 
   describe '.plot' do
+    use_given_filesystem
+
     it 'sends joined parsed options to python script' do
       allow(described_class).to receive(:process_options).and_return(%w{ --test 1 --no-blah })
       allow(described_class).to receive(:plot_helper).and_return('mescript')
       expect(described_class).to receive(:system).with('python mescript 42 --test 1 --no-blah')
       described_class.plot(42, foo: 1, bar: 2)
     end
+
+    it 'saves image file in given location' do
+      dir = given_directory
+      allow(described_class).to receive(:process_options).and_return(%W{--output spec/data/burndown_dir --location #{dir} --no-head})
+      described_class.plot(2, foo: 1)
+      expect(File.exist?(File.join(dir, 'burndown02.png'))).to be true
```

Comment:
does this mean that test is creating the Image? Should we maybe delete it? I am not sure if that happens with other tests. :thinking:

Reply:
Yes, we are creating the image from our test data that already exists because then only we can test that image would be saved in the right place.
As ,I am mocking the parameters to point '--location' to the temp directory I dont think we need to do anything else as file wont exist after test run.

Also, I did this because this is how we are testing the functionality of '--output' option to  check that data file (.YAML) are saved in correct directory.

https://github.com/openSUSE/trollolo/blob/4f2df0497e6fa6e168eed7cc17d849335312ccda/spec/unit/cli_spec.rb#L14-L20